src/ai/backend/agent/kubernetes/agent.py

```diff
@@ -1024,6 +1025,13 @@ async def pull_image(
         # TODO: Add support for appropriate image pulling mechanism on K8s
         pass
 
+    async def purge_images(
+        self,
+        images: list[str],
+    ) -> PurgeImageResponses:
+        # TODO: Add support for appropriate image purging mechanism on K8s
+        return PurgeImageResponses([])
```

Comment:
I think response type should be removed from the internal logic, but we'll work on that in the next refactoring.